### ISSUE TITLE
Add the `no-ambiguous-navigable-target-names` rule

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#1254]
❌ Incorrect
```html
<a href="path/to" target="blank">link</a>

<iframe name="blank"></iframe>
```

✅ Correct
```html
<a href="path/to" target="_blank">link</a>
```
